samples/manager/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "nodejs-docs-samples-iot-manager",
+  "version": "0.0.1",
+  "description": "Example of Cloud IoT device administration",
+  "main": "manager.js",
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "scripts": {
+    "test": "mocha system-test/*.test.js --timeout=600000",
+    "system-test": "mocha system-test/*.test.js --timeout=600000"
+  },
+  "dependencies": {
+    "@google-cloud/iot": "^1.6.1",
+    "@google-cloud/pubsub": "^1.2.0",
+    "googleapis": "^47.0.0",
+    "mocha": "^7.0.0",
+    "uuid": "^3.3.2",
+    "yargs": "^15.1.0"
+  },
+  "cloud-repo-tools": {
```

Comment:
Should be able to yank the `cloud-repo-tools` config section

Reply:
Done.